build/Dockerfile

```diff
@@ -120,6 +120,23 @@ if [[ -d \$LLMDBENCH_RUN_EXPERIMENT_HARNESS_DIR ]]; then \n\
     pushd /workspace/\$LLMDBENCH_RUN_EXPERIMENT_HARNESS_DIR\n\
   fi\n\
   git checkout \$LLMDBENCH_HARNESS_GIT_BRANCH\n\
+  case \${1} in\n\
```

Comment:
Better use `${LLMDBENCH_RUN_EXPERIMENT_HARNESS_DIR}` instead of `${1}`

Reply:
DONE.

*Question:* `${LLMDBENCH_RUN_EXPERIMENT_HARNESS_DIR}` may match `${1}*`, but the git directory names are hardcoded at the top with the initial git clone. In which scenario could it be different than `${1}`?